helios-services/src/main/java/com/spotify/helios/master/MasterService.java

```diff
@@ -206,8 +214,15 @@ public MasterService(final MasterConfig config,
     // and wrap it in our KafkaSender.
     final KafkaSender kafkaSender = new KafkaSender(kafkaClientProvider.getDefaultProducer());
 
+    // Make GooglePubsub senders
+    final List<GooglePubSubSender> pubSubSenders = googlePubSubProvider.senders();
+
+    final List<EventSender> eventSenders = Stream.concat(
+        Stream.of(kafkaSender),
+        pubSubSenders.stream()).collect(Collectors.toList());
```

Comment:
again, this follows the existing coding style, but I think it would make more sense if this was simply an empty list if neither the kakfa sender or pubsub sender were enabled - or a list of just one element if just one of them was enabled - rather than always having two instances which internally have to check if they are enabled or not.

(it is much simpler to simply never call a KafkaSender if kafka-sending is not enabled than to call a KafkaSender that has to wrap it's logic in `!= null` checks)